default/stringtables/en.txt

```diff
@@ -7935,57 +7943,113 @@ Launch Bay
 '''Launch bays are the access point for [[encyclopedia FIGHTER_TECHS]] to enter or exit a ship.
-Each launch bay allows a number of fighters to launch each combat round.
-'''
+Launched fighters join combat the round after being launched, returning once combat ends for that turn.'''
 
 # Ship part classes
 PC_SHIELD
 Shields
 
+PC_SHIELD_DESC
+'''Shield parts reduce the [[encyclopedia DAMAGE_TITLE]] sustained from each weapon hit by the shield parts strength.
+
+Unlike a planet shield, a ship shield is a reflective shield and does not sustain any loss in strength from an attack. Ship shields are ineffective against [[encyclopedia FIGHTER_TECHS]].'''
+
 # Ship part classes
 PC_ARMOUR
 Armor
 
+PC_ARMOUR_DESC
+Armor parts expand the [[metertype METER_STRUCTURE]] of a ship, allowing a ship to sustain more damage before being destroyed.
+
 # Ship part classes
 PC_TROOPS
 Ground Troops
 
+PC_TROOPS_DESC
+'''Troop parts increase the [[metertype METER_TROOPS]] on-board, which can be used to invade enemy planets.
+
+To successfully invade a planet, the total number of [[metertype METER_TROOPS]] from all invading ships must exceed the total defending (ties are awarded to the defender).
+An invading ship is destroyed as part of the landing operation (any additional parts on the ship are lost as well).'''
+
 # Ship part classes
 PC_DETECTION
 Detection
 
+PC_DETECTION_DESC
+'''Detection parts increase the [[metertype METER_DETECTION]] for the ship they are equipped on.
+
+An objects [[metertype METER_STEALTH]] must fall within the empires [[encyclopedia DETECTION_TITLE]] value to be visible.'''
+
 # Ship part classes
 PC_STEALTH
 Stealth
 
+PC_STEALTH_DESC
+'''Stealth parts increase the [[metertype METER_STEALTH]] of a ship. The effects from these parts do not complement each other, only the highest rated part will affect the ship.
+
+If combat occurs in a system, ships that are invisible to an enemy are not targetted by their weapons. If the ship attacks, it may become visible for the following rounds.
+
+See also: [[metertype METER_DETECTION]], [[encyclopedia DETECTION_TITLE]]'''
+
 # Ship part classes
 PC_FUEL
 Fuel
 
+PC_FUEL_DESC
+'''Fuel parts affect a ships [[metertype METER_FUEL]]. Such parts might increase the maximum fuel capacity or increase the amount of fuel the ship can scavenge or regenerate.
+
+A ship requires a fuel value of 1.0 or more to leave a system.'''
+
 # Ship part classes
 PC_COLONY
 Colonization
 
+PC_COLONY_DESC
+'''Colony parts integrate with a ship to provide the necessary equipment to establish ownership on a new planet.
+These parts may also increase the [[metertype METER_POPULATION]] for a particular [[encyclopedia ENC_SPECIES]] on a ship, which will be used to start a colony for that species.
+
+A ship lacking full colony support will still have enough personnel to establish an [[encyclopedia OUTPOSTS_TITLE]].
+(Outposts may construct any available species colony, this combination is cheaper than a populated colony ship)
```

Comment:
colony production requires supply access to the species, on planets meeting other criteria

, to .